src/_premake_init.lua

```diff
@@ -754,8 +754,11 @@
 		allowed = {
 			"Default",
 			"C++98",
+			"C++0x",
 			"C++11",
+			"C++1y",
 			"C++14",
+			"C++1z",
```

Comment:
Unless I'm mistaken, don't these all have `gnu++` equivalents that should be supported too?

Reply:
Dun fixed it.